src/lib/pickles/step.ml

```diff
@@ -179,8 +179,23 @@ struct
           }
         in
         let env =
+          let module Env_bool = struct
+            type t = bool
+
+            let true_ = true
+
+            let false_ = false
+          end in
+          let module Env_field = struct
+            include Tick.Field
```

Comment:
Maybe not worth the effort as it is not super long, but since this only differs with the one later in the file in the Tick vs Tock, perhaps we want to abstract this away